From 842c7b8121b9fa9c39ee97b54a6528135f1aba1c Mon Sep 17 00:00:00 2001
From: Erik Fleckstein <erik@tinkerforge.com>
Date: Tue, 30 Nov 2021 11:52:01 +0100
Subject: [PATCH] LittleFS: Use default implementation of ::exists().

This fixes the log spam when calling LittleFS::exists() on a
non-existing file.
---
 libraries/LittleFS/src/LittleFS.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/libraries/LittleFS/src/LittleFS.cpp b/libraries/LittleFS/src/LittleFS.cpp
index 9ea2320e..b4531c7e 100644
--- a/libraries/LittleFS/src/LittleFS.cpp
+++ b/libraries/LittleFS/src/LittleFS.cpp
@@ -36,19 +36,19 @@ class LittleFSImpl : public VFSImpl
 public:
     LittleFSImpl();
     virtual ~LittleFSImpl() { }
-    virtual bool exists(const char* path);
+    //virtual bool exists(const char* path);
 };
 
 LittleFSImpl::LittleFSImpl()
 {
 }
-
+/*
 bool LittleFSImpl::exists(const char* path)
 {
     File f = open(path, "r",false);
     return (f == true);
 }
-
+*/
 LittleFSFS::LittleFSFS() : FS(FSImplPtr(new LittleFSImpl())), partitionLabel_(NULL)
 {
 }
-- 
2.33.1

